File: php/src/com/example/tool/ToolB.php
<?php

namespace com\example\tool;
require_once 'com/example/tool/ToolA.php';

class ToolB
{
    public static function methodB($logger)
    {
        ToolA::methodA($logger);
        $logger->info("Method B in ToolB called.");
    }
}

File: python/src/python/com/example/tool/ToolB.py
from m8test_java.com.m8test.script.core.api.logger.Logger import Logger
from python.com.example.tool.ToolA import ToolA


class ToolB:
    def methodB(self, logger: Logger):
        toolA = ToolA()
        toolA.methodA(logger)
        logger.info("Method B in ToolB called.")
